portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -20,12 +20,14 @@
   private final String content;
   private final long timestamp;
   private final String ip;
+  private final String imageUrl;
 
-  public Comment(long id, String content, long timestamp, String ip) {
+  public Comment(long id, String content, long timestamp, String ip, String imageUrl) {
```

Comment:
If you change this constructor here, I think you'll need to change where you call it in DataServlet.java as well (every PR should be functional).

If you don't want to implement all the functionality as once (i.e. keep your PR small) then you could just set it to null for now and include a TODO e.g. 

```
// TODO: use the actual image URL when upload functionality is complete.
 comments.add(new Comment(id, content, timestamp, ip, /*imageUrl=*/ null));
```

Note that when you put `null` as an argument to the function, it is good practice to describe what that parameter is with a comment.